### ISSUE TITLE
[1.0] fetchResource:  Cannot read property 'page-component---xxx' of undefined

### ISSUE THREAD

**Issue by 0x80** [gatsbyjs/gatsby#1161]
The following error seems to happen only in development. Maybe it's something you can ignore, because everything seems to work.

```
loader.js:80 Uncaught TypeError: Cannot read property 'page-component---src-templates-work-page-js' of undefined
    at fetchResource (loader.js:80)
    at createResourceDownload (loader.js:38)
    at startResourceDownloading (prefetcher.js:15)
```

Which points to

```javascript
 // Find resource
    var resourceFunction = resourceName.slice(0, 6) === `page-c` ? asyncRequires.components[resourceName] : asyncRequires.json[resourceName];
```

**Comment by KyleAMathews** [comment]
Is this off HEAD? I'm pretty sure I broke some things in dev yesterday which I'll be fixing in a bit.

**Comment by 0x80** [comment]
Yes this was off HEAD

**Comment by KyleAMathews** [comment]
Fixed in #1164 thanks for reporting!
